js/src/components/paid-ads/asset-group/asset-group-card.scss

```diff
@@ -0,0 +1,74 @@
+.gla-asset-group-card {
+	.components-external-link {
+		margin-left: $grid-unit-05;
+	}
+
+	&__error-list {
+		width: 100%;
+		margin: 0;
+		line-height: $gla-line-height-smaller;
+		font-size: $gla-font-smaller;
+		color: $alert-red;
+
+		&:not(:last-child) {
+			margin-bottom: $grid-unit-20;
+		}
+
+		> li {
+			list-style: disc inside;
+			margin: 0;
+			padding-left: 0.5em;
+
+			&:only-child {
+				list-style: none;
+				padding-left: 0;
+			}
+		}
+	}
+}
+
+.gla-asset-field-call-to-action {
+	.components-base-control {
+		width: 240px;
```

Comment:
💅  Maybe we can create a new dimension inside `js/src/css/abstracts/_variables.scss` or even use `$gla-width-small` if it's not very problematic to get that 10px extra.

Reply:
The `$gla-width-small` is a generic name that does not semantically fit the purpose here. Currently, there is only this component that needs this width value for aligning the `<input>` and `<select>`. I tend to move it to **_variables.scss** when there is a more wide reuse.